SU-SWS/jumpstart_ui
# D8CORE-972: Ensure object-fit model.

# READY FOR REVIEW

# Summary
- Ensures the height of the media container and wrappers are always 100% so the object-fit css property can fill the space with the image.

# Needed By (Date)
- Would be nice if it was before the break.

# Urgency
- Low

# Steps to Test

1.  Create a new banner paragraph on a stanford_page with a wide image
2. Add LOTS of text to the hero body so that the overlay is tall
3. Save and review on several breakpoint sizes. The media area should show grey as the background.
4. Check out this branch and clear caches.
5. see background go away and hero image fill available space.

# Affected Projects or Products
- Jumpstart UI
- D8CORE

# Associated Issues and/or People
- D8CORE-972

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
